oeo.owl

```diff
@@ -0,0 +1,230 @@
+Prefix: : <http://openenergy-platform.org/ontology/oeo/>
```

Comment:
In general, documents with Manchester syntax should have filename suffix `omn`.

Reply:
right, I'll fix that